src/main/java/tasks/Task.java

```diff
@@ -0,0 +1,62 @@
+package tasks;
+
+/**
+ * This class is the parent to all the types of Tasks.
+ */
+public class Task {
+    protected String description;
+    protected boolean done;
+
+    /**
+     * Constructor of Task class.
+     * @param description
+     */
+    public Task(String description) {
+        this.description = description;
+        this.done = false;
+    }
+
+    /**
+     * This method gets the description of the Task.
+     * @return The description of the Task.
+     */
+    public String getDescription() {
+
+        return description;
+    }
+
+    /**
+     * This method changes the isDone field to true.
+     */
+    public void markDone() {
+        done = true;
+    }
+
+    /**
+     * This method changes the isDone field to false.
+     */
+    public void markUndone() {
+
+        done = false;
```

Comment:
Inconsistent line breaks. markDone() has no line breaks in the function while markUndone() does. Not a code violation, but a bit strange.